gravity_upgrade.md

```diff
@@ -72,25 +73,126 @@ A manual upgrade operation starts with an operation plan which is a tree of acti
 
 ## Gravity Upgrade Status, Logs, and more
 
+In order to monitor the cluster, it's useful to go through logs and status messages provided by the platform.
+
+Below there's a list of useful commands/resources to use during debugging.
+
+The list is divided based on when the specific command will be more likely used, like during upgrade or at runtime.
+
+### DURING INSTALL / UPGRADE
+  + `sudo ./gravity plan` - from the installer directory
+    This command is used to analyze the progress through any Gravity operations.
+    It's particularly useful to verify at which step if an operation gets stuck,
+    and thus analyze the root cause and to resume once fixed.
+  + content of `gravity-system.log` and (if present) `gravity-install.log` files.
+    These files are usually found in the installation directory
+    or in newer versions they will be located in /var/log/gravity.
+    These files store the output that Gravity generates through installation and
+    it's mostly useful during troubleshooting.
+  + `sudo ./gravity status` - from the installer directory
+    The `status` command output shows the current state of Gravity clusters with
+    any hints about possible causes and hints of a degraded state.
```

Comment:
I would rephrase the second part to: ...with a high level overview of a cluster's health, hints if the cluster is in a degraded state, and the status of individual components.